### PR TITLE
Flatten operator directories and align with Keda module frontrunner

### PR DESCRIPTION
**Pull request by barchw**
**Description**

Changes proposed in this pull request:

- Align our module with Keda frontrunner - move operator directory to repository root
- Add makefiles for module building

**Related issue(s)**

- https://github.com/kyma-project/istio/issues/82

**Test infra PR**
- https://github.com/kyma-project/test-infra/pull/7090

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ifndef MODULE_VERSION
```
Comment (triffer) [inline .env:1]: Isn't it confusing that we have defaults defined in the Makefile itself and the .env file?
What's the benefit for us?

**Review by triffer — APPROVED** [review]
Good job, thanks 👍

File: .env (hunk 1 above)
Comment (barchw) [reply .env:1]: You're right, it might be a little confusing, we can adapt that if we want to

**Pull request merged**